packages/plugin-rows/src/editor/Separator.tsx

```diff
@@ -25,23 +25,21 @@ const AddTrigger = styled.div<{ inline: boolean; name: string }>(
       width: '26px',
       height: '26px',
       borderRadius: '13px',
-      border: `2px solid ${theme.backgroundColor}`,
       display: 'flex',
       justifyContent: 'center',
       alignItems: 'center',
-      color: theme.backgroundColor,
-      backgroundColor: theme.lightBackgroundColor,
+      color: theme.color,
+      backgroundColor: theme.backgroundColor,
       opacity: inline ? 0.6 : 0,
-      transition: '250ms all ease-in-out',
+      transition: '250ms all ease-in-out 250ms',
       position: inline ? 'absolute' : 'relative',
-      zIndex: 70,
+      zIndex: 70, //TODO: check this
```

Comment:
Still open?

Reply:
works fine, can you remove the annotation?